client/model.go

```diff
@@ -125,6 +126,7 @@ type Template struct {
 	Type                 string           `json:"type"`
 	GithubInstallationId int              `json:"githubInstallationId"`
 	UpdatedAt            string           `json:"updatedAt"`
+	TerraformVersion	 string			  `json:"terraformVersion"`
```

Comment:
the indentation here looks strange, but in VS it looks fine.

Reply:
Move to webstorm